src/pages/Calendar/filterCompletedTasks.tsx

```diff
@@ -0,0 +1,9 @@
+import { Tasks } from "../../data/completedTasks";
```

Comment:
You can move this to the utils folder and it can be a ts file as it doesn't return any tsx.

Reply:
File moved and changed to ts